File: KingdomCodeTool/wwwroot/Download/202501201119/Angular/shared/Sys_DataTypes.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_DataTypes } from './Sys_DataTypes.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_DataTypesService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'DataType_Id', 'DataType_Code', 'DataType_Group', 'Description', 'Enabled', 'Remark', 'Creation_Date', 'Created_By', 'Last_Update_Date', 'Last_Updated_By'];

    List: Sys_DataTypes[] | undefined;
    ListFilter: Sys_DataTypes[] | undefined;
    FormData!: Sys_DataTypes;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_DataTypes";
    }
}

File: KingdomCodeTool/wwwroot/Download/202501201119/Service/Implement/Sys_DataTypesService.cs
namespace Service_eHospital_DongNai_A_System.Implement
{
    public class Sys_DataTypesService : BaseService<Sys_DataTypes, ISys_DataTypesRepository>
    , ISys_DataTypesService
    {
    private readonly ISys_DataTypesRepository _Sys_DataTypesRepository;
    public Sys_DataTypesService(ISys_DataTypesRepository Sys_DataTypesRepository) : base(Sys_DataTypesRepository)
    {
    _Sys_DataTypesRepository = Sys_DataTypesRepository;
    }
    }
    }

